utils/test-fixit-apply

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# utils/test-fixit-apply - test fixit-all remapping files -*- python -*-
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+# -----------------------------------------------------------------------------
+"""
+This file is basially a wrapper of llvm migration test command:
+  c-arcmt-test <remaps> | arcmt-test -verify-transformed-files <results>
+
+Clean up files in the system temporary directory generated by `c-arcmt-test`.
+
+Note: This script needs both `c-arcmt-test` and `arcmt-test` are locatable via
+PATH environment variable.
+"""
+# -----------------------------------------------------------------------------
+
+import argparse
+import os
+import subprocess
+import sys
+
+
+# Return true if `path` *looks like* in system temporary directory.
+def is_tmpfile(path):
+    # FIXME: We cannot use tempfile.gettempdir() here because lit doesn't
+    # propagate TMPDIR environment variable.
```

Comment:
👍 Yeah, we don't need to implement a wrapper in the first place.
`env TMPDIR=%t.out c-arcmt-test`